scripts/game.js

```diff
@@ -8,9 +8,10 @@ const { YELLOW, BLUE, RED, GREEN } = constants;
 const colors = [RED, BLUE, YELLOW, GREEN];
 
 class Game {
-  constructor() {
-    this.audioHandler = new AudioHandler();
-    this.board = new Board(this.audioHandler, this.onColorClick.bind(this));
+  constructor({ Board }) {
+    this.board = new BoardHandler({
+      board: new Board(this.onColorClick.bind(this))
```

Comment:
se ve cagado que board tenga una propiedad board, aunque la neta no se me ocurre otra forma. 
A menos que en vez de `this.board` tengamos `this.boardHandler` 🤔